src/structs.rs

```diff
@@ -3,21 +3,26 @@ use std::collections::HashMap;
 
 /** Represents the Dockerfile main stage */
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Default)]
+#[serde(deny_unknown_fields)]
```

Comment:
Can't we just show a warning in the console instead of made the cli fail ?

Reply:
We can't show warning with this lib. We might add it later as a feature